src/h264rtpdepacketizer.cpp

```diff
@@ -10,85 +10,76 @@
 
 #include "h264rtpdepacketizer.hpp"
 #include "nalunit.hpp"
-#include "track.hpp"
 
-#include "impl/logcounter.hpp"
-
-#include <cmath>
-#include <utility>
-
-#ifdef _WIN32
-#include <winsock2.h>
-#else
-#include <arpa/inet.h>
-#endif
+#include "impl/internals.hpp"
 
 namespace rtc {
 
-const unsigned long stapaHeaderSize = 1;
-const auto fuaHeaderSize = 2;
+const binary naluStartCode = {byte{0}, byte{0}, byte{1}};
```

Comment:
What is the rationale for using the 3-byte start code here instead of the 4-byte one?

Reply:
I was wondering if there were any compatibility issues with the 3 vs. 4 byte start sequence, so I googled it and the 3 byte start sequence seemed to be more common for H264, at least on the pages that I looked at :-)

If there are no real compatibility concerns it seems like it would be better to make the H264/H265 de-packetizers consistent.

Reply:
Made it configurable in 27f238fc3c3513cdbbe63fde64beeb18ccf5a326 like https://github.com/paullouisageneau/libdatachannel/pull/1134.